tools/customLinters/rubocop_only_allowed_characters.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'linter_constants'
+
+module CustomCops
+  class OnlyAllowedCharacters < RuboCop::Cop::Base
+    include LinterConstants
+
+    MSG = NOT_ALLOWED_MSG
```

Comment:
wow, I had no idea the Cop was depending on `MSG` being defined! it looks better to define it explicitly like this